lib/diagnostic.rb

```diff
@@ -35,7 +44,7 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
+dave = Person.new('Dave', 32, 'Somerville')
```

Comment:
The instructions were to instantiate a new `Person` with a particular set of properties and then modify them afterward.